repos/fdbt-site/src/interfaces/typeGuards.ts

```diff
@@ -163,9 +166,18 @@ export const isPeriodExpiry = (
     periodExpiryAttribute: PeriodExpiry | ErrorInfo[] | undefined,
 ): periodExpiryAttribute is PeriodExpiry => !!periodExpiryAttribute && 'productValidity' in periodExpiryAttribute;
 
+export const isDayOfTheWeek = (dayOfTheWeek: DayOfTheWeek | ErrorInfo[] | undefined): dayOfTheWeek is DayOfTheWeek =>
+    !!dayOfTheWeek;
+
+export const isCapStart = (capStart: CapStart | ErrorInfo[] | undefined): capStart is CapStart => !!capStart;
```

Comment:
We can remove this for the above reasons also - this function is just checking if something is defined.